frontend/pages/review.tsx

```diff
@@ -15,16 +15,18 @@ import { useTheme } from '@mui/material/styles'
 import Tab from '@mui/material/Tab'
 import Tabs from '@mui/material/Tabs'
 import Typography from '@mui/material/Typography'
+import { useGetSchema } from 'data/schema'
 import Link from 'next/link'
 import React, { useState } from 'react'
+import DisabledElementTooltip from 'src/common/DisabledElementTooltip'
 import { getErrorMessage } from 'utils/fetcher'
 
 import { postEndpoint } from '../data/api'
 import { ApprovalCategory, ApprovalFilterType, useGetNumApprovals, useListApprovals } from '../data/approvals'
```

Comment:
This isn't necessarily for you to fix, but I've noticed that we have a duplicate type here. We should be using the type in `bailo/frontend/types/types.ts`. If that type needs to change to no longer be an enum, then we should do it in that file. 

@ARADDCC002 any insight on what's happened here?